docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -274,11 +345,11 @@ version: v3
 ```
 
 </TabItem>
-<TabItem label="Uploaded JSON file">
+<TabItem label="Upload JSON">
 
-Use this method for single self-hosted Teleport Auth instances, or when you can
-easily and reliably make the JSON file available to all hosts running the Auth
-service.
+The following command uploads the JSON file to a host running the Teleport Auth Service.
```

Comment:
The command does **not** upload the JSON file to the host, it only assumes that it's available on that path in the auth host.